src/cartesian_filter.cpp

```diff
@@ -0,0 +1,34 @@
+#include <tallies/box_position_filter.hpp>
+#include <tallies/cartesian_filter.hpp>
+#include <tallies/regular_cartesian_mesh_filter.hpp>
+#include <utils/constants.hpp>
+
+CartesianFilter::CartesianFilter(Position r_low, Position r_high)
+    : r_low_(r_low), r_high_(r_high) {
+  if ((r_low_.x() > r_high_.x()) || (r_low_.y() > r_high_.y()) ||
+      (r_low_.z() > r_high_.z()))
+    fatal_error(
+        " Corrdinates of \"low\" position are higher than \"high\" "
+        "position.\n");
+}
+
+// make the cartesian filter
+std::shared_ptr<CartesianFilter> make_cartesian_filter(const YAML::Node& node) {
+  std::shared_ptr<CartesianFilter> cartesian_filter_ = nullptr;
+
+  if (!node["Position-Filter"]) {
```

Comment:
Keys in the yaml file should be lower case.
```suggestion
  if (!node["position-filter"]) {
```